src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -79,17 +79,18 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
     {
       LM_W(("Bad Input (No entity id specified"));
       eP->errorCode.fill("ParseError", "no entity id specified");
-      ciP->httpStatusCode = SccBadRequest;;
+      ciP->httpStatusCode = SccBadRequest;
+
       return eP->render(ciP, EntitiesRequest);
     }
   }
-  else if (document.Empty()) 
```

Comment:
I looked for that info. But without any luck. The documentation on rapidjson is not very good.
It might be Empty works on vectors and ObjectEmpty on objects?
Sorry, I don't know.

What I **do** know is that there are plenty of complaints about "very small json docs and crashes" on the internet and especially that the broker crashes using Empty, but not when using ObjectEmpty ...